packages/compiler/src/__tests__/__snapshots__/typescript.test.ts.snap

```diff
@@ -650,12 +454,12 @@ Object {
       "isStatic": false,
     },
     "kind": "enum",
-    "members": Array [
-      Object {
-        "defaultValue": "\\"danger\\"",
+    "members": [
+      {
+        "defaultValue": undefined,
```

Comment:
default values are getting dropped - not sure why 🤔

Reply:
found out that this is due to a [breaking change in v0.23.0](https://typedoc.org/guides/changelog/#v0.23.0-(2022-06-26)):

> `reflection.defaultValue` is no longer set for enum members. The same information is available on `reflection.type` with more precision.

will fix!

Reply:
✅ fixed